pavelib/paver_tests/test_paver_quality.py

```diff
@@ -132,6 +133,74 @@ def test_generic_value_none_found(self):
         actual_count = pavelib.quality._get_count_from_last_line(self.f.name, "foo")  # pylint: disable=protected-access
         self.assertEqual(actual_count, None)
 
+    def test_get_safelint_counts_happy(self):
```

Comment:
Updated class level docstring, as well as adding docstrings for individual test functions.